src/main/java/ee/carlrobert/codegpt/actions/IncludeFilesInContextAction.java

```diff
@@ -93,11 +93,6 @@ public void nodeStateChanged(@NotNull CheckedTreeNode node) {
   }
 
   private @Nullable FileCheckboxTree getCheckboxTree(DataContext dataContext) {
-    var psiElement = CommonDataKeys.PSI_ELEMENT.getData(dataContext);
```

Comment:
I removed this, since if you can right click anywhere in the file your cursor might be at a method declaration, Class name or similiar and then this would return the corresponding `PsiElement` which then tries to find the corresponding file for it, instead of the currently open file.
Is there any context where this is used @carlrobertoh? From what I can see the `IncludeFilesInContextAction` is only added to the files context-menu, for which the  `VIRTUAL_FILE_ARRAY.getData(dataContext);` works.

Reply:
> Is there any context where this is used @carlrobertoh? From what I can see the IncludeFilesInContextAction is only added to the files context-menu, for which the VIRTUAL_FILE_ARRAY.getData(dataContext); works.

Good question. I think that for older IDE builds, the `VIRTUAL_FILE_ARRAY` didn't exist when selecting a single file, so I had to have two separate implementations. However, I believe this is no longer the case since I bumped the 'since' version a while ago.